apps/docs/pages/guides/platform/going-into-prod.mdx

```diff
@@ -54,10 +54,31 @@ After developing your project and deciding it's Production Ready, you should run
 - Supabase employs a number of safeguards against bursts of incoming traffic to prevent abuse and help maximize stability across the platform
   - If you're expecting high load events including production launches or heavy load testing, or prolonged high resource usage please give us at least 2 weeks notice. You can do this by opening a ticket via the [support form](https://app.supabase.com/support/new).
 
+### Rate Limits
+
+- The table below shows the rate limit quotas on the following authentication endpoints:
+
+| Endpoint                                         | Path                                                           | Limited By               | Rate Limit                                                                      |
+| ------------------------------------------------ | -------------------------------------------------------------- | ------------------------ | ------------------------------------------------------------------------------- |
+| All endpoints that send emails                   | `/auth/v1/signup` `/auth/v1/recover` `/auth/v1/user`[^1]       | Sum of combined requests | Defaults to 30 emails per hour. Is customizable with custom SMTP set up.        |
+| All endpoints that send One-Time-Passwords (OTP) | `/auth/v1/otp`                                                 | Sum of combined requests | Defaults to 30 OTPs per hour. Is customizable.                                  |
+| Send OTPs or magiclinks                          | `/auth/v1/otp`                                                 | Last request             | Defaults to 60 seconds window before a new request is allowed. Is customizable. |
+| Signup confirmation request                      | `/auth/v1/signup`                                              | Last request             | Defaults to 60 seconds window before a new request is allowed. Is customizable. |
+| Password Reset Request                           | `/auth/v1/recover`                                             | Last request             | Defaults to 60 seconds window before a new request is allowed. Is customizable. |
+| Verification requests                            | `/auth/v1/verify`                                              | IP Address               | 360 requests per hour (with bursts up to 30 requests)                           |
+| Token refresh requests                           | `/auth/v1/token`                                               | IP Address               | 360 requests per hour (with bursts up to 30 requests)                           |
+| Create or Verify an MFA challenge                | `/auth/v1/factors/:id/challenge` `/auth/v1/factors/:id/verify` | IP Address               | 15 requests per minute (with bursts up to 30 requests)                          |
```

Comment:
What does the version in the path refer to? Could be confusing depending on the client library I'm using.

```suggestion
| All endpoints that send emails                   | `/auth/v1/signup` `/auth/v1/recover` `/auth/v1/user`[^1]       | Sum of combined requests | Defaults to 30 emails per hour. Is customizable with custom SMTP setup.        |
| All endpoints that send One-Time-Passwords (OTP) | `/auth/v1/otp`                                                 | Sum of combined requests | Defaults to 30 OTPs per hour. Is customizable.                                  |
| Send OTPs or magiclinks                          | `/auth/v1/otp`                                                 | Last request             | Defaults to 60-second window before a new request is allowed. Is customizable. |
| Signup confirmation request                      | `/auth/v1/signup`                                              | Last request             | Defaults to 60-second window before a new request is allowed. Is customizable. |
| Password reset request                           | `/auth/v1/recover`                                             | Last request             | Defaults to 60-second window before a new request is allowed. Is customizable. |
| Verification requests                            | `/auth/v1/verify`                                              | IP Address               | 360 requests per hour (with bursts up to 30 requests).                           |
| Token refresh requests                           | `/auth/v1/token`                                               | IP Address               | 360 requests per hour (with bursts up to 30 requests).                           |
| Create or Verify an MFA challenge                | `/auth/v1/factors/:id/challenge` `/auth/v1/factors/:id/verify` | IP Address               | 15 requests per minute (with bursts up to 30 requests).                          |
```